src/main/kotlin/net/developerden/devdenbot/listener/JoinLeaveListener.kt

```diff
@@ -15,8 +17,18 @@ import net.dv8tion.jda.api.events.guild.member.GuildMemberRemoveEvent
 class JoinLeaveListener : EventListener {
     private suspend fun onGuildMemberJoin(event: GuildMemberJoinEvent) {
         val channel = event.jda.getTextChannelById(WELCOME_CHANNEL_ID)
+        val memberCount = event.guild.memberCount
+        val member = event.member
+        val guild: Guild = event.guild
+        val role: Role? = guild.getRoleById(666) // TODO ALEX please put the role idea for 666 :)
```

Comment:
i'm kinda relucant to add easter eggs like this just because whilst they're cool, they'll just lead to more people wanting funny gimmick roles. the meme roles were a similar idea, but in the end they just look messy. 
also, this could lead to multiple people getting the role, which probably isn't what we want